src/server/mod.rs

```diff
@@ -20,6 +19,11 @@ use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
 mod connection;
 mod waiting_users;
 
+pub struct User {
```

Comment:
`WaitingUser` is now just `User` since we also use this struct in `connection.rs` now.